test/integration/scheduler/volume_binding_test.go

```diff
@@ -431,10 +433,12 @@ func testVolumeBindingStress(t *testing.T, schedulerResyncPeriod time.Duration,
 		}
 	}
 
+	klog.Infof("Start creating PVs and PVCs")
 	// Create enough PVs and PVCs for all the pods
-	pvs := []*v1.PersistentVolume{}
-	pvcs := []*v1.PersistentVolumeClaim{}
-	for i := 0; i < podLimit*volsPerPod; i++ {
+	podVolumesCount := podLimit * volsPerPod
+	pvs := make([]*v1.PersistentVolume, podVolumesCount)
+	pvcs := make([]*v1.PersistentVolumeClaim, podVolumesCount)
+	workqueue.ParallelizeUntil(context.TODO(), 16, podVolumesCount, func(i int) {
```

Comment:
I like this idea - to pre-init an array and also avoid data racing in a concurrent calculation.